src/python/CRABInterface/RESTTask.py

```diff
@@ -115,6 +115,16 @@ def webdir(self, **kwargs):
         yield row.user_webdir
 
 
+    def getpublishurl(self, **kwargs):
+        if 'workflow' not in kwargs or not kwargs['workflow']:
+            raise InvalidParameter("Task name not found in the input parameters")
+        workflow = kwargs['workflow']
+        try:
+            row = next(self.api.query(None, None, self.Task.GetPublishUrl_sql, taskname=workflow))
+        except StopIteration:
+            raise ExecutionError("Impossible to find task %s in the database." % kwargs["workflow"])
+        return row
```

Comment:
does kwargs[""workflow"] really work ? Or dictionary key needs to be in single quotes kwargs['workflow'] ?  In any case consistent syntax will make it easier to read.